packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -8,6 +8,20 @@ import 'package:flutter/painting.dart';
 
 export 'dart:ui' show TextDirection;
 
+/// Determines the assertiveness level of the accessibility announcement.
+///
+/// It is used to set the priority with which assistive technology should treat announcements.
```

Comment:
nit: include a link to `AnnounceSemanticsEvent` or `AnnounceSemanticsEvent.assertivenessSetting` as a breadcrumb in this doc to give the reader some context what this is used for.